src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -372,6 +436,67 @@ g.test('formats')
     t.expectOK(result);
   });
 
+g.test('default_blend_color,initial_blend_color')
+  .desc(`Test that the initial blend color is correctly set at the beginning of every sub passes.`)
+  .fn(async t => {
+    const kFormat = 'rgba8unorm';
+    const kSize = 64;
+    const kWhiteColorData = new Float32Array([255, 255, 255, 255]);
+    const kBlackColorData = new Float32Array([0, 0, 0, 0]);
+
+    const basePipeline = t.createRenderPipelineForTest(kFormat, undefined);
+    const testPipeline = t.createRenderPipelineForTest(kFormat, {
+      srcFactor: 'constant',
+      dstFactor: 'one',
+      operation: 'add',
+    });
+
+    const renderTarget = t.device.createTexture({
+      usage: GPUTextureUsage.RENDER_ATTACHMENT | GPUTextureUsage.COPY_SRC,
+      size: [kSize, kSize],
+      format: kFormat,
+    });
+
+    const commandEncoder = t.device.createCommandEncoder();
+    const renderPass = commandEncoder.beginRenderPass({
+      colorAttachments: [
+        {
+          view: renderTarget.createView(),
+          loadOp: 'load',
+          storeOp: 'store',
+        },
+      ],
+    });
+    renderPass.setPipeline(basePipeline);
+    renderPass.setBindGroup(
+      0,
+      t.createBindGroupForTest(basePipeline.getBindGroupLayout(0), kBlackColorData)
+    );
+    renderPass.draw(3);
+    renderPass.setPipeline(testPipeline);
+    renderPass.setBindGroup(
+      0,
+      t.createBindGroupForTest(testPipeline.getBindGroupLayout(0), kWhiteColorData)
+    );
+    renderPass.draw(3);
```

Comment:
Comment to explain the progression of the test.
```suggestion
    renderPass.draw(3);
    // Draw [1,1,1,1] with `src * constant + dst * 1`.
    // The blend constant defaults to [0,0,0,0], so the result is
    // `[1,1,1,1] * [0,0,0,0] + [0,0,0,0] * 1` = [0,0,0,0].
```

Reply:
Added.